client/my-sites/subscribers/subscriber-list/styles.scss

```diff
@@ -0,0 +1,125 @@
+@import "@automattic/color-studio/dist/color-variables";
+@import "@wordpress/base-styles/breakpoints";
+
+.subscriber-list {
+	margin: 0;
+
+	.row {
+		border-block-end: 1px solid $studio-gray-5;
+		display: flex;
+		align-items: center;
+		flex-direction: row;
+		padding-top: 20px;
+		padding-bottom: 20px;
+
+		* {
+			flex: 1;
+		}
+
+		.hidden {
+			display: none;
+		}
+
+		&.header {
+			padding-bottom: $font-code;
+			padding-top: 0;
+
+			@media (max-width: $break-small) {
+				display: none;
+			}
+		}
+
+		.subscriber-list__checkbox-column,
+		.subscriber-list__profile-column,
+		.subscriber-list__subscription-type-column,
+		.subscriber-list__rate-column,
+		.subscriber-list__since-column,
+		.subscriber-list__menu-column {
+			font-weight: 400;
+			font-size: $font-body-small;
+			font-size: rem(14px);
+			line-height: 20px;
+			letter-spacing: -0.15px;
+			color: $studio-gray-60;
+		}
+
+		.subscriber-list__profile-column {
+			display: flex;
+			align-items: center;
+			flex: 3;
+			min-width: 0;
+
+			.subscriber-profile {
+				display: flex;
+				min-width: 0;
+
+				.subscriber-profile__user-image {
+					height: 40px;
+					width: 40px;
+					border-radius: 50%;
+					margin-right: 12px;
+					flex: 0;
+				}
+
+				.subscriber-profile__user-details {
+					display: flex;
+					flex-direction: column;
+					justify-content: center;
+					max-width: 100%;
+					overflow: hidden;
+					min-width: 0;
+
+					.subscriber-profile__name {
+						overflow: hidden;
+						text-overflow: ellipsis;
+						white-space: nowrap;
+						font-weight: bold;
+						line-height: 17px;
+						min-width: 0;
+						font-size: rem(15px);
```

Comment:
Here we could use font-size variable instead (from `@import "@automattic/typography/styles/variables";`)